static/js/modules/nootils.js

```diff
@@ -19,8 +19,8 @@ module.exports = function(settings) {
       } else {
         result = new Date();
       }
-      result.setUTCDate(result.getUTCDate() + days);
```

Comment:
I honestly can't work out how this would be different to what I changed it to, since days are days, and the rules around wrapping are presumably as good here as they are there. I changed it to be more "default".